x-pack/plugins/ingest_manager/public/applications/ingest_manager/sections/agent_config/index.tsx

```diff
@@ -15,6 +15,9 @@ export const AgentConfigApp: React.FunctionComponent = () => (
       <Route path="/configs/:configId/add-datasource">
         <CreateDatasourcePage />
       </Route>
+      <Route path="/configs/:configId/:tabId">
```

Comment:
@nchaulet quick question: were you also suggesting that I not add the sub-route path here? and instead initialize a new `Router` down at the details level to capture sub-route paths?

I made the change to use `<Route>`, but left this alone for now. Let me know.

Reply:
@paul-tavares yes I think you should not capture `:tabId` here and use Route in the details page with react-router 5 you do not have to declare a new router for that just the nested routes

Reply:
cool. Will refactor it to do that.